src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -67,19 +66,11 @@ class Filter extends Component<Props> {
       agg => agg.slice === "MAJOR_PERIOD"
     )
 
-    const hasLabFeature =
-      user &&
-      user.lab_features &&
-      user.lab_features.includes("New Buy Now Flow")
-    const enableBuyNowFlow = sd.ENABLE_NEW_BUY_NOW_FLOW || hasLabFeature
-
     return (
       <>
         <Flex flexDirection="column" alignItems="left" mt={-1} mb={1}>
           {!hideTopBorder && <Separator mb={1} />}
-          {enableBuyNowFlow
-            ? this.renderWaysToBuy(filterState, mediator, counts)
-            : this.renderForSaleCheckbox(filterState, mediator, counts)}
+          {this.renderWaysToBuy(filterState, mediator, counts)}
```

Comment:
can `renderForSaleCheckbox()` get removed in that case? (minor nit- a linter probably is underlining that if its unused, but we dont actually run/enforce that in CI).

Reply:
@mzikherman - this would get flagged and prevent a push unless `--no-verify` is used: https://github.com/artsy/reaction/blob/master/tsconfig.json#L12 and https://github.com/artsy/reaction/blob/master/package.json#L35

Reply:
Oh actually, renderForSaleCheckbox is a method. There's no real way to check that methods aren't being used -- that's why its good to avoid `this`.